app/queries/applied_coupons_query.rb

```diff
@@ -22,7 +22,7 @@ def base_scope
   end
 
   def with_coupon_code(scope)
-    scope.where(coupons: {code: filters.coupon_code})
+    scope.joins(:coupon).where(coupons: {code: filters.coupon_code})
```

Comment:
How did this work before 🤯

Reply:
It was working because of https://github.com/getlago/lago-api/pull/3687/files#diff-0ee28d6592941883dd2f0006587a46e2d9e66ef444ab746cbf3db6a778c6fc60L38 but it's not anymore :)